src/vivarium_gates_child_iv_iron/results_processing/process_results.py

```diff
@@ -142,14 +167,24 @@ def aggregate_over_seed(data: pd.DataFrame) -> pd.DataFrame:
     ], axis=1).reset_index()
 
 
-def pivot_data(data: pd.DataFrame) -> pd.DataFrame:
-    return (
-        data
-        .set_index(GROUPBY_COLUMNS)
-        .stack()
-        .reset_index()
-        .rename(columns={f'level_{len(GROUPBY_COLUMNS)}': 'key', 0: 'value'})
-    )
+def pivot_data(data: pd.DataFrame, disaggregate_seeds: bool) -> pd.DataFrame:
+    if disaggregate_seeds:
+        nonstandard_groupby_cols = GROUPBY_COLUMNS + [results.RANDOM_SEED_COLUMN]
```

Comment:
Can reduce copy/paste jobs by creating a local groupby_cols object in the `if` and then doing this work on that.

eg
```
if disaggregate_seeds:
    groupby_cols = GROUPBY_COLUMNS + [results.RANDOM_SEED_COLUMN]
else:
    groupby_cols = GROUPBY_COLUMNS
return (
    data
    .set_index(groupby_cols)
    ...
)

Reply:
This seems a lot better.